{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/static_source/sass/base/mixins/_breakpoint.scss

```diff
@@ -0,0 +1,18 @@
+@mixin breakpoint($point) {
+   @if $point == desktop {
```

Comment:
This is a really neat idea but in practice you often times want control over the size or to control it in an easier way via the setting.